backend/app/helpers/message_helpers.py

```
@@ -21,14 +24,30 @@
 
 
 def create_system_message(
-    user_id: Optional[str] = None, user_name: Optional[str] = None
+    user_id: Optional[str] = None,
```

Ruff (0.14.8)

Unused function argument: `user_id`

(ARG001)